### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by bansalnitish**
Just learning Pull request

### CODE REVIEW

**Comment by ankita240796** [comment]
I can comment

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # TextEditor
-Minimalist Text Editor
+Minimalist Text Editor to edit plain text
```
Comment (bansalnitish) [inline README.md]: yo

Comment (ankita240796) [reply README.md]: yo

File: README.md [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # TextEditor
```
Comment (ankita240796) [inline README.md:1]: yo yo

File: texteditor.py [hunk 3]
```diff
@@ -394,3 +394,5 @@ def about():
 window.protocol('WM_DELETE_WINDOW',close1)
```
Comment (bansalnitish) [inline texteditor.py:394]: yo

**Pull request closed without merging**